### DISCUSSION TITLE
Custom logger

### DISCUSSION THREAD

**Discussion by sawa-ko** [TypedProject/ts-express-decorators#2055] [Q&A] [answered]
I see that tsed has its own logger, but the project where I work has its own logger as well. How would it be to make tsed use the logger that the project already has?

Something like:
```ts
@Configuration({ logger: { instance: new MyCustomLogger() } })
export class Server {}
```

**Comment by Romakita** [comment, marked as answer]
Hello @kaname-png 

Use @tsed/logger-connect to register your own log. See the documentation here: 

https://logger.tsed.io/appenders/connect.html#configuration

See you
Romain

**Reply by Romakita** [reply]
> logger that the project already has

Ok which project? which company?
